pandas/tests/util/test_assert_almost_equal.py

```diff
@@ -454,6 +455,19 @@ def test_assert_almost_equal_shape_mismatch_override():
         tm.assert_almost_equal(np.array([1, 2]), np.array([3, 4, 5]), obj="Index")
 
 
+@pytest.mark.parametrize(
+    "obj",
+    [
+        MockNumpyLikeArray(np.ndarray((2,) * 1)),
```

Comment:
should this be `np.array` instead of `np.ndarray`?  weird way to call the constructor, and the result here also looks iterable